packages/eslint-plugin-svelte/src/rules/no-invalid-html-elements.ts

```diff
@@ -0,0 +1,54 @@
+import type { AST } from 'svelte-eslint-parser';
+import { createRule } from '../utils';
+import { getSourceCode } from '../utils/compat';
+
+const INVALID_HTML_ELEMENTS = ['head', 'body', 'window', 'document', 'element', 'options'];
+const VALID_PREFIX = 'svelte:';
+
+export default createRule('no-invalid-html-elements', {
+	meta: {
+		docs: {
+			description: 'Disallows valid Svelte 4 tags, that no are no longer valid in Svelte 5',
+			category: 'Possible Errors',
+			// TODO: Switch to recommended in the major version
+			recommended: false
+		},
+		schema: [],
+		messages: {
+			invalidElement: 'Invalid {{name}} element, use svelte:{{name}} instead.'
+		},
+		type: 'problem', // 'problem', or 'layout',
+		fixable: 'code'
+	},
+	create(context) {
+		const sourceCode = getSourceCode(context);
+		const ctx = sourceCode.parserServices.svelteParseContext;
+		if (!(ctx?.compilerVersion ?? '').startsWith('5')) {
```

Comment:
I don't think this branch is necessary in my opinion. `<svelte:*>` is available in v4 too. I think it makes sense for people to use the new way even if they're using v4. What do you think?

Reply:
Yes, that's true, the only thing is that the elements are _valid_ in Svelte 4, for example. That's why I preferred to report an error only in Svelte 5 or higher, where such elements are not valid.

Reply:
Hmm. How about renaming the rule to encourage its use even in Svelte v4? How about naming it `no-deprecated-raw-special-elements`?

Reply:
And would we report it as an error for v4, or something more as a warning?

Reply:
For now, I think it's user preference.
I'm not sure yet how we'll be warned if we add new rules to the sharable config.
If we add a sharable configuration for v4, we might be able to warn when we use it.